kernel/classes/datatypes/ezmatrix/ezmatrixtype.php

```diff
@@ -39,13 +39,16 @@ function validateObjectAttributeHTTPInput( $http, $base, $contentObjectAttribute
         $data = false;
         if ( $http->hasPostVariable( $base . '_ezmatrix_cell_' . $contentObjectAttribute->attribute( 'id' ) ) )
             $data = $http->PostVariable( $base . '_ezmatrix_cell_' . $contentObjectAttribute->attribute( 'id' ) );
+	    if ( is_array( $data ) )
+	    {
         $count = 0;
         for ( $i = 0; $i < count( $data ); ++$i )
              if ( trim( $data[$i] ) <> '' )
              {
                  ++$count;
                  break;
              }
+	     }
```

Comment:
$count could be not defined in line 52
```suggestion
        $count = 0;
	if ( is_array( $data ) )
	{
            for ( $i = 0; $i < count( $data ); ++$i )
                 if ( trim( $data[$i] ) <> '' )
                 {
                     ++$count;
                     break;
                 }
        }
```

Reply:
Hello @blankse I have made the requested changes.